### ISSUE TITLE
pub

### ISSUE THREAD

**Issue by mscroggs** [FEniCS/web#2]
There's a lot of stuff currently at https://fenicsproject.org/pub/

Some of this could me moved into this repo.

**Comment by mscroggs** [comment]
In particular, we want to be sure that https://fenicsproject.org/pub/photos/ are not lost

**Comment by mscroggs** [comment]
I have local copy of all of /pub on my local hard drive in case anything is lost when we switch over.

**Comment by chrisrichardson** [comment]
I think we should do some filtering before adding all to git

**Comment by mscroggs** [comment]
Yes. The full files are 7.7GB and many of them (eg tars of old versions) aren't really the kind of files that are best hosted on GitHub (perhaps Google Drive/Dropbox/similar instead if we want to rehost them)

**Comment by jhale** [comment]
Amazon S3 bucket or similar might be best.
